korman/properties/modifiers/gui.py

```diff
@@ -474,7 +482,19 @@ def pre_export(self, exporter, bo):
         else:
             rgn_obj = self.clickable_region
 
-        yield self.convert_logic(bo, age_name=exporter.age_name, version=exporter.mgr.getVer(), region=rgn_obj)
+        # Generate the same six foot cube if we need a share region.
```

Comment:
It's not quite six feet, and the comment should probably be closer to the cube generator ;)

Reply:
Edited the comment and moved it just above the `create_cube` function.